app/views/events/_apply_button.erb

```diff
@@ -0,0 +1,7 @@
+<% if current_user.present? && current_user.application_letter_for_event?(event) %>
+    <%= link_to t("helpers.links.show_application"), check_application_letter_path(current_user.application_letter_for_event(event)),
```

Comment:
From the [user story description](https://github.com/hpi-swt2/workshop-portal/issues/244):

> When a user applied for an event the "Bewerben" buttons should be replace with "Bewerbung anzeigen" and link to the application checking page.